compose/config/config.py

```diff
@@ -734,7 +733,7 @@ def validate_service(service_config, service_names, config_file):
             "Service '{name}' contains uppercase characters which are not valid "
             "as part of an image name. Either use a lowercase service name or "
             "use the `image` field to set a custom name for the service image."
-            .format(name=service_name))
+                .format(name=service_name))
```

Comment:
Please undo this indentation, since it's not related to the PR. It also triggers:
> PEP 8: continuation line unaligned for handling indent